rero_ils/modules/loans/utils.py

```diff
@@ -35,12 +35,21 @@ def get_circ_policy(loan):
     patron = Patron.get_record_by_pid(loan.get('patron_pid'))
     patron_type_pid = patron.patron_type_pid
 
-    return CircPolicy.provide_circ_policy(
+    result = CircPolicy.provide_circ_policy(
         library_pid,
         patron_type_pid,
         holding_circulation_category
     )
 
+    # checkouts and request are not allowed anymore for blocked patrons
+    if patron.get('blocked', False):
+        result.update({
+            "allow_checkout": False,
+            "allow_requests": False,
+        })
+
+    return result
```

Comment:
I'm little bit confused with that. This is not the cipo that disallow chekout/request but the Patron module.

Reply:
Patron module only gives the information about a blocked patron.
And circulation policy behaves regarding this information.
This permit to the UI to not change a lot to work.